code/modules/unit_tests/simple_animal_freeze.dm

```diff
@@ -0,0 +1,421 @@
+		/mob/living/simple_animal/hostile/skeleton/ice,
+		/mob/living/simple_animal/hostile/skeleton/plasmaminer,
+		/mob/living/simple_animal/hostile/skeleton/plasmaminer/jackhammer,
+		/mob/living/simple_animal/hostile/skeleton/templar,
+		/mob/living/simple_animal/hostile/smspider,
+		/mob/living/simple_animal/hostile/smspider/overcharged,
+		/mob/living/simple_animal/hostile/space_dragon,
+		/mob/living/simple_animal/hostile/syndicate,
+		/mob/living/simple_animal/hostile/syndicate/civilian,
+		/mob/living/simple_animal/hostile/syndicate/mecha_pilot,
+		/mob/living/simple_animal/hostile/syndicate/mecha_pilot/nanotrasen,
+		/mob/living/simple_animal/hostile/syndicate/mecha_pilot/no_mech,
+		/mob/living/simple_animal/hostile/syndicate/mecha_pilot/no_mech/nanotrasen,
+		/mob/living/simple_animal/hostile/syndicate/melee,
+		/mob/living/simple_animal/hostile/syndicate/melee/space,
+		/mob/living/simple_animal/hostile/syndicate/melee/space/stormtrooper,
+		/mob/living/simple_animal/hostile/syndicate/melee/sword,
+		/mob/living/simple_animal/hostile/syndicate/melee/sword/space,
+		/mob/living/simple_animal/hostile/syndicate/melee/sword/space/stormtrooper,
+		/mob/living/simple_animal/hostile/syndicate/ranged,
+		/mob/living/simple_animal/hostile/syndicate/ranged/infiltrator,
+		/mob/living/simple_animal/hostile/syndicate/ranged/shotgun,
+		/mob/living/simple_animal/hostile/syndicate/ranged/shotgun/space,
+		/mob/living/simple_animal/hostile/syndicate/ranged/shotgun/space/stormtrooper,
+		/mob/living/simple_animal/hostile/syndicate/ranged/smg,
+		/mob/living/simple_animal/hostile/syndicate/ranged/smg/orion,
+		/mob/living/simple_animal/hostile/syndicate/ranged/smg/pilot,
+		/mob/living/simple_animal/hostile/syndicate/ranged/smg/space,
+		/mob/living/simple_animal/hostile/syndicate/ranged/smg/space/stormtrooper,
+		/mob/living/simple_animal/hostile/syndicate/ranged/space,
+		/mob/living/simple_animal/hostile/syndicate/ranged/space/stormtrooper,
+		/mob/living/simple_animal/hostile/syndicate/space,
+		/mob/living/simple_animal/hostile/syndicate/space/stormtrooper,
+		/mob/living/simple_animal/hostile/tree,
+		/mob/living/simple_animal/hostile/tree/festivus,
+		/mob/living/simple_animal/hostile/vatbeast,
+		/mob/living/simple_animal/hostile/venus_human_trap,
+		/mob/living/simple_animal/hostile/viscerator,
+		/mob/living/simple_animal/hostile/wizard,
+		/mob/living/simple_animal/hostile/zombie,
+		/mob/living/simple_animal/mouse,
+		/mob/living/simple_animal/mouse/brown,
+		/mob/living/simple_animal/mouse/brown/tom,
+		/mob/living/simple_animal/mouse/cable_lover,
+		/mob/living/simple_animal/mouse/gray,
+		/mob/living/simple_animal/mouse/white,
+		/mob/living/simple_animal/parrot,
+		/mob/living/simple_animal/parrot/natural,
+		/mob/living/simple_animal/parrot/poly,
+		/mob/living/simple_animal/parrot/poly/ghost,
+		/mob/living/simple_animal/pet,
+		/mob/living/simple_animal/pet/cat,
+		/mob/living/simple_animal/pet/cat/_proc,
```

Comment:
not a fan of you one bit

Reply:
https://adw.org/about-us/resources/request-an-exorcism/

Reply:
jesus